src/System.Numerics.Vectors/tests/Performance/HashCodeTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xunit.Performance;
+using Xunit.Sdk;
+
+namespace System.Numerics.Performance.Tests
+{
+    public class HashCodeTest
+    {
+        private static int BaseHashCodeCombine(int h1, int h2)
+        {
+            return ((h1 << 5) + h1) ^ h2;
+        }
+
+        [Benchmark(InnerIterationCount = 1000)]
+        public void BaseHashTest()
```

Comment:
This comparison isn't apples-to-apples, it's apples to _almost_ apples.  You need to either run the second test against vectors before the modifications and compare the output (what I would do), or extract the logic into a separate method here as well (which potentially cares too much about the internals of `GetHashCode()`).
Performance tests are often more "meta" results, and not really intended for keeping comparisons for all types indefinitely.